test/selenium/lib/view/signed_out/accessors.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var webdriver = require('selenium-webdriver');
+var By = require('selenium-webdriver').By;
+
+var SELECTORS = Object.freeze({
+    page: By.id('signin')
+});
+
+function SignedOutAccessor(driver) {
+  this.driver = driver;
+};
+
+SignedOutAccessor.prototype = {
+   get isSignOutPage(){
+   	  return this.driver.isElementPresent(SELECTORS.page);
```

Comment:
:warning: We should make sure the element is also displayed. If I remember the DOM correctly, the whole page is present, and we just toggle the `hidden` attribute.